src/components/custom/dashboard/shared/SidebarDashboard.jsx

```
@@ -57,168 +57,170 @@ const ProfileCapsule = ({
+        className={`p-3 cursor-pointer group transition-all duration-200 hover:bg-white/[0.02] rounded-xl ${
+          isCollapsed ? "flex justify-center" : ""
         }`}
+        onClick={onProfileClick}
+        tabIndex="0"
+        onKeyDown={(e) => {
+          if (e.key === "Enter" || e.key === " ") {
+            e.preventDefault();
+            onProfileClick?.();
+          }
+        }}
       >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)